Nightly copr builds are broken

**Issue Description**
After 2022-01-27 nightly builds started to fail during srpm generation step:
```
npm ERR! code EPERM
npm ERR! syscall spawn
npm ERR! errno -1
npm ERR! Error: spawn EPERM
npm ERR!     at ChildProcess.spawn (node:internal/child_process:413:11)
npm ERR!     at spawn (node:child_process:700:9)
npm ERR!     at /usr/lib/node_modules/npm/node_modules/@npmcli/promise-spawn/index.js:36:12
npm ERR!     at new Promise (<anonymous>)
npm ERR!     at promiseSpawnUid (/usr/lib/node_modules/npm/node_modules/@npmcli/promise-spawn/index.js:35:13)
npm ERR!     at promiseSpawn (/usr/lib/node_modules/npm/node_modules/@npmcli/promise-spawn/index.js:15:10)
npm ERR!     at runScriptPkg (/usr/lib/node_modules/npm/node_modules/@npmcli/run-script/lib/run-script-pkg.js:54:13)
npm ERR!     at runScript (/usr/lib/node_modules/npm/node_modules/@npmcli/run-script/lib/run-script.js:9:16)
npm ERR!     at Array.<anonymous> (/usr/lib/node_modules/npm/node_modules/@npmcli/arborist/lib/arborist/rebuild.js:320:17)
npm ERR!     at run (/usr/lib/node_modules/npm/node_modules/promise-call-limit/index.js:30:26)
npm ERR!  Error: spawn EPERM
npm ERR!     at ChildProcess.spawn (node:internal/child_process:413:11)
npm ERR!     at spawn (node:child_process:700:9)
npm ERR!     at /usr/lib/node_modules/npm/node_modules/@npmcli/promise-spawn/index.js:36:12
npm ERR!     at new Promise (<anonymous>)
npm ERR!     at promiseSpawnUid (/usr/lib/node_modules/npm/node_modules/@npmcli/promise-spawn/index.js:35:13)
npm ERR!     at promiseSpawn (/usr/lib/node_modules/npm/node_modules/@npmcli/promise-spawn/index.js:15:10)
npm ERR!     at runScriptPkg (/usr/lib/node_modules/npm/node_modules/@npmcli/run-script/lib/run-script-pkg.js:54:13)
npm ERR!     at runScript (/usr/lib/node_modules/npm/node_modules/@npmcli/run-script/lib/run-script.js:9:16)
npm ERR!     at Array.<anonymous> (/usr/lib/node_modules/npm/node_modules/@npmcli/arborist/lib/arborist/rebuild.js:320:17)
npm ERR!     at run (/usr/lib/node_modules/npm/node_modules/promise-call-limit/index.js:30:26) {
npm ERR!   errno: -1,
npm ERR!   code: 'EPERM',
npm ERR!   syscall: 'spawn'
npm ERR! }
npm ERR! 
npm ERR! The operation was rejected by your operating system.
npm ERR! It is likely you do not have the permissions to access this file as the current user
npm ERR! 
npm ERR! If you believe this might be a permissions issue, please double-check the
npm ERR! permissions of the file and its containing directories, or try running
npm ERR! the command again as root/Administrator.
```

`copr-rpmbuild` executes mock with `mock-source-build.cfg` config that drops a lot of capabilities: https://pagure.io/copr/copr/blob/main/f/rpmbuild/mock-source-build.cfg#_9

I've installed `bcc-tools` and monitored `npm ci` with `/usr/share/bcc/tools/capable` to detect what capabilities they use. `CAP_SETGID` was the one that caused `EPERM`.

Turns out, this is a behaviour change in NPM since 7.x:
https://github.com/npm/cli/blob/latest/changelogs/CHANGELOG-7.md#all-lifecycle-scripts

> The user, group, uid, gid, and unsafe-perms configurations are no longer relevant. When npm is run as root, scripts are always run with the effective uid and gid of the working directory owner.

Files had `root:mockbuild` ownership, and `npm` was running under root, so `npm` tried to set gid to `mockbuild` user and was denied.

We should always chown files in $CWD to `root:root` since the `make srpm` step is executed under root user

--- a/.copr/Makefile
+++ b/.copr/Makefile
@@ -6,6 +6,13 @@ srpm:
 	# Install build dependencies
 	dnf install -y dnf-plugins-core
 	dnf builddep -y --skip-broken --spec rpmbuild/SPECS/389-ds-base.spec --best --allowerasing --setopt=install_weak_deps=False
+	# chown files in current working directory to root:root
+	# because when npm is run as root, scripts are always run
+	# with the effective uid and gid of the working directory owner.
+	# copr-rpmbuild runs mock with CAP_SETUID and CAP_SETGID
+	# capabilities dropped, and build commands are executed as root.
+	# So npm fails if current working directory is not owned by root.
+	chown -R root:root .
 	# Generate srpm
 	SKIP_AUDIT_CI=1 make -f rpm.mk srpms
 	

